docs/wiki/EL/JSON-RPC.md

```diff
@@ -0,0 +1,113 @@
+| engine_getPayloadBodiesByHashV1*         |           block_hash (array)           | given block hashes returns bodies of the corresponding execution payloads |
+| engine_getPayloadV1*                     |  forkchoice_state, payload attributes  | obtains execution payload from payload build process                      |
+| debug_newPayloadV1*                      |                tx_hash                 | returns execution payload validation                                      |
+
+Those methods marked with an asterisk (*) have more than one version. The [Ethereum JSON-RPC specification](https://ethereum.github.io/execution-apis/api-documentation/) provides a detailed description.
+
+### Eth
+
+This one is a particularly one of the most used. Just a brief list of the methods is provided here, but the full list can be found in the [Ethereum JSON-RPC specification](https://ethereum.github.io/execution-apis/api-documentation/).
+
+| **Method**               |      **Params**       | **Description**                                                 |
+|--------------------------|:---------------------:|-----------------------------------------------------------------|
+| eth_blockNumber          |  no mandatory params  | returns the number of the most recent block                     |
+| eth_call                 |  transaction object   | executes a new message call immediately without creating a transaction on the block chain |
+| eth_chainId              |  no mandatory params  | returns the current chain id                                     |
+| eth_estimateGas          |  transaction object   | makes a call or transaction, which won't be added to the blockchain and returns the used gas, which can be used for estimating the used gas |
+| eth_gasPrice             |  no mandatory params  | returns the current price per gas in wei                         |
+| eth_getBalance           |  address, block number | returns the balance of the account of the given address          |
+| eth_getBlockByHash       |  block hash, full txs  | returns information about a block by hash                        |
+| eth_getBlockByNumber     |  block number, full txs | returns information about a block by block number                |
+| eth_getBlockTransactionCountByHash | block hash | returns the number of transactions in a block from a block matching the given block hash |
+| eth_getBlockTransactionCountByNumber | block number | returns the number of transactions in a block from a block matching the given block number |
+| eth_getCode              |  address, block number | returns code at a given address in the blockchain                 |
+| eth_getLogs              |  filter object         | returns an array of all logs matching a given filter object       |
+| eth_getStorageAt         |  address, position, block number | returns the value from a storage position at a given address |
+
+## Tooling
```

Comment:
Before diving into tooling, I would explain that client can support different protocols for communication with the client - http, wss or ipc.